src/pid.cpp

```diff
@@ -81,16 +81,28 @@ Pid::Pid(const Pid & source)
 Pid::~Pid() {}
 
 void Pid::initPid(
-  double p, double i, double d, double i_max, double i_min, NodeParamsIfacePtr node_param_iface)
+  double p, double i, double d, double i_max, double i_min, NodeParamsIfacePtr node_param_iface,
```

Comment:
why is the `node_param_iface` and the `parameter_prefix` not part of the constructor?